wemake_python_styleguide/visitors/ast/builtins.py

```diff
@@ -99,10 +99,11 @@ def _check_is_magic(self, node: ast.Num) -> None:
         self.add_violation(MagicNumberViolation(node, text=str(node.n)))
 
     def _check_is_approximate_constant(self, node: ast.Num) -> None:
-        if node.n in constants.APPROXIMATE_CONSTANTS:
-            self.add_violation(
-                ApproximateConstantViolation(node, text=str(node.n)),
-            )
+        for constant in constants.APPROXIMATE_CONSTANTS:
+            if abs(constant - node.n) < 0.004:
```

Comment:
Let's move `0.004` to `constants.py` as well. And change it to `0.0001`

Reply:
I think 0.0001 is too small. pi - 3.14 is smaller than 0.0001 :(

Reply:
Ok, then `0.01`

Reply:
Done! :)